# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../gq/gq-1.2.2-auth.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Index: branches/gq-1-2/ChangeLog
===================================================================
--- ./ChangeLog	(revision 1149)
+++ ./ChangeLog	(revision 1150)
@@ -1,3 +1,8 @@
+2007-05-14  Sven Herzberg  <herzi@gnome-de.org>
+
+	* src/util.c: fix some connectivity issues (based on observation and
+	patch from Pawel Salek on gqclient-discuss)
+
 # 1.2.2
 
 2006-11-13  Sven Herzberg  <herzi@gnome-de.org>
Index: branches/gq-1-2/src/util.c
===================================================================
--- ./src/util.c	(revision 1149)
+++ ./src/util.c	(revision 1150)
@@ -532,6 +532,7 @@
 	switch(gtk_dialog_run(GTK_DIALOG(glade_xml_get_widget(xml, "password_dialog")))) {
 	case GTK_RESPONSE_OK:
 		g_free_and_dup(server->bindpw, gtk_entry_get_text(GTK_ENTRY(glade_xml_get_widget(xml, "input_password"))));
+		string = g_strdup (server->bindpw);
 
 		// FIXME: store password only from successful connections
 		if(gnome_keyring_is_available() &&
